src/loganalytics/azext_loganalytics/_params.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+def load_arguments(self, _):
+    with self.argument_context('loganalytics') as c:
+        c.argument('workspace', options_list=['--workspace', '-w'], help='GUID of the Log Analytics Workspace', required=True)
+        c.argument('kql', help='Query to execute over Log Analytics data.', required=True)
```

Comment:
you don't need the `required=True` for workspace or kql; they will be implied from the command signature in `execute_query`